src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -698,35 +704,57 @@ sub-<label>/[ses-<label>/]
+| PhaseEncodingDirection | REQUIRED              | [string][]    | See [in-plane spatial encoding](#in-plane-spatial-encoding) table of fields.       |
+| TotalReadoutTime       | REQUIRED              | [number][]    | See [in-plane spatial encoding](#in-plane-spatial-encoding) table of fields.       |
+
+For example:
 
 ```JSON
 {
    "PhaseEncodingDirection": "j-",
-   "TotalReadoutTime": 0.095,
-   "IntendedFor": "func/sub-01_task-motor_bold.nii.gz"
+   "TotalReadoutTime": 0.095
 }
 ```
 
-The `label` value of the [`dir-<label>`](../99-appendices/09-entities.md#dir) key/value pair
-can be set to arbitrary alphanumeric label (`[a-zA-Z0-9]+` for
-example `LR` or `AP`) that can help users to distinguish between different
-files, but should not be used to infer any scanning parameters (such as phase
-encoding directions) of the corresponding sequence. Please rely only on the JSON
-file to obtain scanning parameters. \_epi files can be a 3D or 4D - in the
-latter case all timepoints share the same scanning parameters. To indicate which
-run is intended to be used with which functional or diffusion scan the
-IntendedFor field in the JSON file should be used.
+As for other EPI sequences, these field mapping sequences may have any of the
+[in-plane spatial encoding](#in-plane-spatial-encoding) metadata keys.
+However, please note that `PhaseEncodingDirection` and `TotalReadoutTime` keys
+are REQUIRED for these field mapping sequences.
+
+### Expressing the MR protocol intent for fieldmaps
+
+#### Using `IntendedFor` metadata
+
+Fieldmap data can be linked to the specific scan(s) it was
+acquired for by filling the `IntendedFor` field in the corresponding JSON file.
+
+| **Key name** | **Requirement level** | **Data type**                         | **Description**                                                                                                                                                                                                                                                                |
+| ------------ | --------------------- | ------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| IntendedFor  | RECOMMENDED           | [string][] or [array][] of [string][] | Contains one or more filenames with paths relative to the participant subfolder. The path needs to use forward slashes instead of backward slashes. This field is OPTIONAL, and in case the fieldmaps do not correspond to any particular scans it does not have to be filled. |
+
+For example:
+
+```JSON
+{
+   "IntendedFor": ["ses-pre/func/sub-01_ses-pre_task-motor_run-1_bold.nii.gz",
+                   "ses-post/func/sub-01_ses-post_task-motor_run-1_bold.nii.gz"]
```

Comment:
This example seems unrealistic. Is there any real-world situation where people would acquire one field map in a longitudinal study and then apply it to fMRI scans from multiple sessions? I would lean toward a more common case.

```suggestion
   "IntendedFor": ["ses-pre/func/sub-01_ses-pre_task-motor_run-1_bold.nii.gz",
                   "ses-pre/func/sub-01_ses-pre_task-motor_run-2_bold.nii.gz"]
```

Reply:
Agreed. Sorry I saw your suggestion too late (and didn't commit your suggestion), but I came up with exactly the same example.